dcs/action.py

```diff
@@ -1825,45 +1857,42 @@ class PictureToAll(PictureAction):
 
     def __init__(self, file_res_key: ResourceKey, seconds: int,
                  clearview: bool, start_delay: int,
-                 horz_alignment: str, vert_alignment: str,
-                 size: int, size_units: str):
+                 horz_alignment: PictureAction.HorzAlignment, vert_alignment: PictureAction.VertAlignment,
+                 size: int, size_units: PictureAction.SizeUnits) -> None:
         super().__init__(predicate=self.predicate, file_res_key=file_res_key,
                          seconds=seconds, clearview=clearview,
                          start_delay=start_delay, horz_alignment=horz_alignment,
-                         vert_alignment=horz_alignment, size=size, size_units=size_units)
+                         vert_alignment=vert_alignment, size=size, size_units=size_units)
         self.params = [self.file_res_key, seconds, clearview, start_delay,
                        horz_alignment, vert_alignment, size, size_units]
 
     @classmethod
-    def create_from_dict(cls, d, mission):
+    def create_from_dict(cls, d, mission) -> "PictureToAll":
         return cls(ResourceKey(d["file"]), d["seconds"], d["clearview"], d["start_delay"],
                    d["horzAlignment"], d["vertAlignment"], d["size"], d["size_units"])
 
-    def dict(self):
-        return super().dict()
-
 
 class PictureToCoalition(PictureAction):
     predicate = "a_out_picture_s"
 
-    def __init__(self, coalition: str, file_res_key: ResourceKey, seconds: int,
+    def __init__(self, coalition: Coalition, file_res_key: ResourceKey, seconds: int,
                  clearview: bool, start_delay: int,
-                 horz_alignment: str, vert_alignment: str,
-                 size: int, size_units: str):
+                 horz_alignment: PictureAction.HorzAlignment, vert_alignment: PictureAction.VertAlignment,
+                 size: int, size_units: PictureAction.SizeUnits) -> None:
         super().__init__(predicate=self.predicate, file_res_key=file_res_key,
                          seconds=seconds, clearview=clearview,
                          start_delay=start_delay, horz_alignment=horz_alignment,
-                         vert_alignment=horz_alignment, size=size, size_units=size_units)
+                         vert_alignment=vert_alignment, size=size, size_units=size_units)
         self.coalition = coalition
         self.params = [self.coalition, self.file_res_key, seconds, clearview,
                        start_delay, horz_alignment, vert_alignment, size, size_units]
 
     @classmethod
-    def create_from_dict(cls, d, mission):
+    def create_from_dict(cls, d, mission) -> "PictureToCoalition":
```

Comment:
Need type annotations. There are a couple others missing later in the file too.

Reply:
Noted.